cirq/google/engine/engine_test.py

```diff
@@ -148,34 +149,39 @@
         '1562544000021',
         'metrics': [{
             'name': 'xeb',
-            'targets': ['q0_0', 'q0_1'],
+            'targets': ['0_0', '0_1'],
             'values': [{
```

Comment:
The current format returned today is qX_Y, but it shouldn't matter. If we'd like to abstract/simplify these, should we collapse to a single value, e.g. "['0', '1']"?

Reply:
Oh.. They shouldn't do that.  It should use the canonicalization defined in GridQubit!

Reply:
Added code to handle first and will file bug.